src/search/view/SearchView.ts

```diff
@@ -151,7 +151,10 @@ export class SearchView extends BaseTopLevelView implements TopLevelView<SearchV
 										m(NavButton, {
 											label: "emails_label",
 											icon: () => BootIcons.Mail,
-											href: () => "/search/mail",
+											href: "#",
+											click: async () => {
```

Comment:
nitpick: I am usually against making `click` or any event handlers async because it confuses people with what it does or does not.

Since you are calling an async function anyway I'd propose to just drop those async/await qualifiers